ipv6/tayga/Makefile

```diff
@@ -3,12 +3,12 @@
 include $(TOPDIR)/rules.mk
 
 PKG_NAME:=tayga
-PKG_VERSION:=0.9.2
-PKG_RELEASE:=5
+PKG_VERSION:=0.9.5
+PKG_RELEASE:=1
 
-PKG_SOURCE:=tayga-$(PKG_VERSION).tar.bz2
-PKG_SOURCE_URL:=http://www.litech.org/tayga/
-PKG_HASH:=2b1f7927a9d2dcff9095aff3c271924b052ccfd2faca9588b277431a44f0009c
+PKG_SOURCE:=$(PKG_VERSION).tar.gz
```

Comment:
This is not right. The tarball will be downloaded as version.tar.gz and this could be overwritten by something else.